index.html

```diff
@@ -0,0 +1,318 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
it doesn't contain the complete documentation, it would be better to always check the code a couple of time from beginning to end